src/controller/CHIPDeviceController.h

```diff
@@ -537,6 +538,26 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
      */
     CHIP_ERROR EstablishPASEConnection(NodeId remoteDeviceId, RendezvousParameters & params);
 
+    /**
+     * @brief
+     *   Start establishing a PASE connection with a node for the purposes of commissioning.
+     *   Commissioners that wish to use the auto-commissioning functions should use the
+     *   supplied "PairDevice" functions above to automatically establish a connection then
+     *   perform commissioning. This function is intended to be use by commissioners that
+     *   are not using the supplied auto-commissioner.
+     *
+     *   This function is non-blocking. PASE is established once the DevicePairingDelegate
+     *   receives the OnPairingComplete call.
+     *
+     *   PASE connections can only be established with nodes that have their commissioning
+     *   window open. The PASE connection will fail if this window is not open and the
+     *   OnPairingComplete will be called with an error.
+     *
+     * @param[in] remoteDeviceId        The remote device Id.
+     * @param[in] setUpCode             The setup code for connecting to the device
+     */
+    CHIP_ERROR EstablishPASEConnection(NodeId remoteDeviceId, const char * setUpCode);
```

Comment:
Please check with @sagar-apple as to which Darwin framework callsites that were depending on the previous behavior need to get switched over to this new function?

Reply:
Talked to @sagar-apple.  We (Apple) will handle whatever changes to `CHIPDeviceController.mm` are needed here, so you don't have to worry about it.

Reply:
They do :) The PairDevice APIs in the CHIPDeviceController.mm need to start calling Establish PASE everywhere instead of the cpp PairDevice APIs.